R/rvar-dist.R

```diff
@@ -59,14 +71,42 @@ cdf.rvar <- function(x, q, ...) {
   })
 }
 
+#' @rdname rvar-dist
+#' @export
+cdf.rvar_factor <- function(x, q, ...) {
+  stop_no_call("cdf() cannot be calculated for rvar_factor() objects.")
```

Comment:
Note to self: to be consistent with distributional::cdf() and ::quantile() on categorical distributions, these should probably return `NA` vectors instead of an error. My experience adding rvar_factor support to ggdist also suggests this would be easier to use than throwing an error as well.

Reply:
done